src/core/StelLocationMgr.cpp

```diff
@@ -491,6 +491,10 @@ StelLocationMgr::StelLocationMgr()
 #endif
 	// Init to Paris France because it's the center of the world.
 	lastResortLocation = locationForString(conf->value("init_location/last_location", "Paris, Western Europe").toString());
+
+	planetName="Earth";
+	planetSurfaceMap.load(StelFileMgr::findFile("data/gui/miscWorldMap.jpg", StelFileMgr::File));
```

Comment:
`MapWidget` initializes its pixmap as `QPixmap(":/graphicGui/miscWorldMap.jpg")`. I suppose you'll want to use the same path here.

Reply:
Ah, did you make install before that test? Thanks, this would have been stupid!

Reply:
Yes, I always make install.